drivers/fprotect/Kconfig

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config FPROTECT
+	bool "Write protect flash regions"
+	depends on SOC_FAMILY_NRF
+	help
+	  Use hardware peripherals (BPROT, ACL, or SPU) to write protect
```

Comment:
Thinking about it, MPU is confusing. I'm referring to the nRF51 peripheral (renamed to BPROT in nRF52), not the (more widely recognized) ARM MPU present on every device. I think we should actually remove it from the list in the docs to reduce confusion, since it shares its implementation with BPROT and nRF51 is old.

Reply:
Removed it from docs now.